opensrp-app/src/main/java/org/smartregister/repository/P2PSenderTransferDao.java

```diff
@@ -25,40 +27,73 @@
 
 public class P2PSenderTransferDao extends BaseP2PTransferDao implements SenderTransferDao {
 
+    private static final String SEPARATOR = "-";
+
+    private P2POptions p2POptions;
+
+    public P2PSenderTransferDao() {
+        this(null);
+    }
+
+    public P2PSenderTransferDao(P2POptions p2POptions) {
```

Comment:
Let's disable this. I just looked at the workflow and we'll have to call

`CoreLibrary.getInstance().getP2POptions().setLocationsFilter(locationsFilter)` everytime before starting the P2P activity. The instance of this class is created at the start of the application at which point we don't know the specific filters

Reply:
Removed